src/main/java/lotto/domain/lotto/CreateRandomNumberStrategy.java

```diff
@@ -0,0 +1,15 @@
+package lotto.domain.lotto;
+
+import java.util.Random;
+
+public class CreateRandomNumberStrategy implements CreateLottoNumberStrategy {
```

Comment:
클래스명은 동사를 사용하지 않는게 관례입니다 :) 명사구로 시작해주세요!
```suggestion
public class RandomNumberStrategy implements CreateLottoNumberStrategy {
```

Reply:
감사합니다! 변경하겠습니다 ㅎ